lib/project_types/script/messages/messages.rb

```diff
@@ -150,11 +150,14 @@ module Messages
           language_library_for_api_not_found_cause: "Script can’t be pushed because the %{language} library for API %{api} is missing.",
           language_library_for_api_not_found_help: "Make sure extension_point.yml contains the correct API library.",
           no_scripts_found_in_app: "The selected apps have no scripts. Please, create them first on the partners' dashboard.",
-          missing_env_file_variables: "The following are missing in the .env file: %s."\
-            " To add it, run {{command:%s script connect}}",
-          missing_push_options: "The following are missing: %s. "\
-            "To add them to a CI environment:\n\t1. Run a connect command {{command:%s script connect}}\n\t2. Navigate to the .env file at the root of your project\n\t"\
-            "3. Copy the missing values, then pass them through as arguments.",
+          missing_push_options_ci: "The following are missing: %s. ",
+          missing_push_options_ci_solution: "To add them to a CI environment:\n\t1. Run a connect command " \
+          "({{command:%1$s script connect}})\n\t2. Navigate to the .env file at the root of your project\n\t" \
+          "3. Copy the missing values and pass them through as arguments in {{command:%1$s script push}}",
+          missing_env_file_variables: "The following are missing in the .env file: %s. ",
+          missing_env_file_variables_solution: "To add it, connect your script with " \
+          "{{command:%1$s script connect}} " \
+          "or run {{command:%1$s script register}} to register a new script.",
```

Comment:
We don't have a `register` command yet

```suggestion
          "{{command:%1$s script connect}}.",
```